### PR TITLE
implemented tag component

### PR DESCRIPTION
**Pull request by aaronkang7**
### Summary 

This pull request implements address tag components. Color for corresponding location tag still needs to be confirmed by designers. 

This pull request is the first step towards implementing feature Foo

- [x] implemented tag feature (made separate component)
- [x] Color hex codes confirmed by designers.

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

**Review by mg2239 — CHANGES_REQUESTED** [review]
tags look good! but by component i meant a separate component, since we also use the tags when displaying ride information on the schedule. could you separate the code out from the table cells? you dont need to add tags to the tables either, since we're reimplementing all the tables this sprint

erika should also be getting back to you about the colors to use, so be sure to add those too 👍

File: frontend/src/components/Tag/Tag.tsx
Comment (mg2239) [inline frontend/src/components/Tag/Tag.tsx:42]: why not styles[tagStyle]?

**Review by mg2239 — CHANGES_REQUESTED** [review]
make sure the right tag styles are being used in the right places, and make sure the tags are positioned correctly

ex. should be not reduced, tag circles should be vertically centered
image

File: frontend/src/components/Tag/Tag.tsx
Comment (aaronkang7) [reply frontend/src/components/Tag/Tag.tsx:42]: oh... i didnt know you could do that ahah

**Review by mg2239 — CHANGES_REQUESTED** [review]
looks good overall, the tag positioning can probably wait until the new table style are merged in. please implement the new tag colors though

**Comment by aaronkang7** [comment]
> looks good overall, the tag positioning can probably wait until the new table style are merged in. please implement the new tag colors though

new tag colors? I believe the colors were confirmed by designers quite a while ago. Were there any design changes that I may not be aware of? 

I found the hex codes from the "Design System" page of the Figma

**Review by mg2239 — APPROVED** [review]
ah yeah i see now, the full sized tag styles weren't implemented correctly so thats why i couldnt see the new colors. just fixed and should be good to go

**Pull request merged**
